extension/resources/walkthrough/view-container.md

```diff
@@ -36,6 +37,6 @@ If DVC and the extension are initialized, side bar will expose multiple views:
   columns.
 - `PLOTS`. Used to fine-tune the plots dashboard, e.g. by toggling plots
   visibility.
-- `FILTER BY`. Used to view, add, or remove metrics and parameters filters.
+- `FILTER BY`. Used to view, add, or remove metrics/parameters filters. These
+  will hide specific experiments in the table.
```

Comment:
Could be wrong but I think the mention about hiding experiments should be in the filter by section. I don't think sorting hides experiments 🤔